app/plugins/hmac-authentication.md

```diff
@@ -16,17 +16,25 @@ nav:
       - label: Signature Authentication Scheme
       - label: Signature Parameters
       - label: Signature String Construction
+      - label: Clock Skew
+      - label: Body Validation
+      - label: Enforcing Headers
       - label: HMAC Example
       - label: Upstream Headers
 ---
 
-Add HMAC Signature Authentication to your APIs to establish the identity of the consumer. The plugin will check for valid signature in the `Proxy-Authorization` and `Authorization` header (in this order). This plugin implementation follows the [draft-cavage-http-signatures-00](https://tools.ietf.org/html/draft-cavage-http-signatures-00) draft with slightly changed signature scheme.
+Add HMAC Signature authentication to your APIs to establish the integrity of
+incoming requests. The plugin will validate the digital signature sent in the
+`Proxy-Authorization` or `Authorization` header (in this order). This plugin
+implementation is based off the [draft-cavage-http-signatures][draft] draft
+with a slightly different signature scheme.
 
 ----
 
 ## Configuration
 
-Configuring the plugin is straightforward, you can add it on top of an [API][api-object] by executing the following request on your Kong server:
+Configuring the plugin is straightforward, you can add it on top of an
```

Comment:
This is how it is mentioned on other plugin pages https://getkong.org/plugins/key-authentication/